link-24-wrapup

baxtan opened the pull request.

baxtan committed:
link-24-wrapup

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+TremensS, tomson, ravel_of, Noor, Maxime, eddyoaa, notfelix, dalrae
+
+Discussions:
+- Laser safety induction by TremenS
```
missing small s before last S

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+Discussions:
+- Laser safety induction by TremenS
+- Potential upcoming workshop by TremenS
```
missing small s before last S

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+Discussions:
+- They got the core functionality working
+- Can draw basic SVG elements, (primitives only)
+- Skia now has a GCode writer node
```
Skia will get a GCodeWriter node

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+Discussions:
+
+- How to simplify Kairos for typical usecases
+- How Kairos and Global Channels work togeter
```
together

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+- Trying to find common UI workflows
+- Kairos as a VL extension progress made by azeno and Natan
+- Currently working on adding a default timeline
+- New nuget coming in the near future
```
already available, link to announcement in vl.kairos chat

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+### 03. **Controlling software**
+
+Natan Sinigaglia, gregsn, David, azeno, texone
```
David Brüll

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+Discussions:
+
+- General interest in bringin gamma build games and experiences into Steam to draw attention to vvvv and Stride in other communitites
```
bringing_ gamma built_ games

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+Discussions:
+
+- Nuget TOR browser support was developed and published during LINK!
```
add link: https://www.nuget.org/packages/VL.TorSharp

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+- How do these two worlds relate to each other?
+- How can we improve inter-operability?
+
+More to that coming soon...
```
i'd remove this sentence

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+workshop ideas for the coming semester:
+
+-  A workshop about controlling lasers by TremenS
```
missing small s before last S

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+workshop ideas for the coming semester:
+
+-  A workshop about controlling lasers by TremenS
+-  A workshop covering teh best practices about Building Applications in vvvv by sebl and ravazquez
```
remove "the" before best practices

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+-  Creative Robots series workshop by amir
+
+
+### Project Shwocase
```
Showcase

joreg commented on content/blog/2024/Dates/link-summercamp-24-wrapup/index.md:
```diff
@@ -0,0 +1,197 @@
+
+Special thanks to sebl who kicked off a series of conversation and project showcases.
+
+sebl, chk, amir, ravazquez, David, Maxime
```
David Brüll

baxtan committed:
- Interspersing all the Haikus with a note at the end

- Corrected typos and added relevant links to topics
- Change the release date to 2024.09.18
